### PR TITLE
Add example of GitHub projects with efffective structure

### LINKED ISSUE #16: Add examples of GitHub projects with efffective organization / structure
Read article: https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#project-structure-and-organization

Add here: https://github.com/mozilla/open-leadership-training-series/blob/gh-pages/_articles/get-your-project-online/project-set-up-for-collaboration-with-github.md

### PR DESCRIPTION
**Pull request by nufail**
Resolves #16

### CODE REVIEW

File: _articles/get-your-project-online/project-set-up-for-collaboration-with-github.md [hunk 1]
```diff
@@ -17,9 +17,26 @@ materials: "A computer, an internet connection, and a free GitHub account"
 
 To work most effectively with contributors, your project structure should be clean and welcoming, with folder titles that make sense. For example, your project might have different folders for data, technical documentation, and images. README.md and CONTRIBUTING.md (which you’ll make in the next module) should be in your root directory, along with your ROADMAP. Think of these documents as an anchor for your project. As you add files, you may want to organize these into folders and create a consistent naming convention. Here’s an example of good project set-up.
 
+[GitHub hubot project structure](https://github.com/github/hubot)
```
Comment (abbycabs) [inline _articles/get-your-project-online/project-set-up-for-collaboration-with-github.md]: This is helpful! Thanks @nufail!

I'd still like some links to a few effectively structured github projects of various sizes / languages.

**Comment by nufail** [comment]
I've added @betatim suggestion and few other language wise examples

**Comment by betatim** [comment]
My preferred solution would be to make a good list of links (we have that now 🎉) and then shorten the structure you show to include only the documents that the previous paragraph refers to. People might look at the `hubot` example and then wonder what all the directories are good for and why they weren't mentioned so far or if they should also have those, etc.

File: _articles/get-your-project-online/project-set-up-for-collaboration-with-github.md [hunk 2]
```diff
@@ -17,9 +17,33 @@ materials: "A computer, an internet connection, and a free GitHub account"
 
 To work most effectively with contributors, your project structure should be clean and welcoming, with folder titles that make sense. For example, your project might have different folders for data, technical documentation, and images. README.md and CONTRIBUTING.md (which you’ll make in the next module) should be in your root directory, along with your ROADMAP. Think of these documents as an anchor for your project. As you add files, you may want to organize these into folders and create a consistent naming convention. Here’s an example of good project set-up.
```
Comment (betatim) [inline _articles/get-your-project-online/project-set-up-for-collaboration-with-github.md]: Could you also fix the english in the last sentence of this paragraph? Should be "Here's an example of **a** good project set-up."

**Comment by nufail** [comment]
Modified according to the feedback. Thanks @betatim

**Comment by betatim** [comment]
LGTM, giving @acabunoc a chance to look before merging. Should we keep #16 open to collect some more example projects?

**Pull request merged**